### ISSUE TITLE
Default KolmogorovSamplingSize is too low

### ISSUE THREAD

**Issue by mbaudin47** [openturns/openturns#1217]
The following test performs the KS test on a sample from a gaussian.
```
import openturns as ot
data = ot.Normal(1).getSample(1000)
fact = ot.NormalFactory()
ksresult = ot.FittingTest.Kolmogorov(data, fact)
```
However, the test performs poorly, because the default parameters are too small:
```
ot.ResourceMap.SetAsUnsignedInteger("FittingTest-KolmogorovSamplingSize",10)
```
As a result, probabilities equal to 0., 0.1, 0.2, ..., 0.9, 1.0 are computed.
In order to make it work, the following parameters can be used:
```
ot.ResourceMap.SetAsUnsignedInteger("FittingTest-KolmogorovSamplingSize",1000)
```
This default setting was done because there are performance issues with the creation of chaos polynomials based on input samples. However, the current setting is counter-intuitive for most users, who use default values and get *completely* wrong results.

**Comment by mbaudin47** [comment]
Furthermore, the doc has issues:
http://openturns.github.io/openturns/1.13/user_manual/_generated/openturns.FittingTest_Kolmogorov.html

* the example in the doc does not show the two cases,
* the default value of the `level` parameter is not known.

Example 1: the parameters are estimated from data
```
import openturns as ot
ot.RandomGenerator.SetSeed(0)
distribution = ot.Normal()
sample = distribution.getSample(30)
factory = ot.NormalFactory()
ot.ResourceMap.SetAsUnsignedInteger("FittingTest-KolmogorovSamplingSize",1000)
ot.FittingTest.Kolmogorov(sample, factory)
```
Case 2: the parameters are known
```
import openturns as ot
ot.RandomGenerator.SetSeed(0)
distribution = ot.Normal()
sample = distribution.getSample(30)
ot.FittingTest.Kolmogorov(sample, distribution)
```
Example 2: the parameters are known
```
import openturns as ot
ot.RandomGenerator.SetSeed(0)
distribution = ot.Normal()
sample = distribution.getSample(30)
ot.FittingTest.Kolmogorov(sample, distribution)
```
Example 3: set the level
```
import openturns as ot
ot.RandomGenerator.SetSeed(0)
distribution = ot.Normal()
sample = distribution.getSample(30)
ot.FittingTest.Kolmogorov(sample, distribution, 0.01)
```

**Comment by mbaudin47** [comment]
Part of the job is done in PR#1265. But the examples still have to be updated within the "Examples" section of FittingTest_Kolmogorov.html

**Comment by mbaudin47** [comment]
Actually, this ticket is a duplicate of #1061: if the fitting test was faster, the current implementation would not require to have a default KolmogorovSamplingSize so low. 

This is why I close the current issue.

**Comment by mbaudin47** [comment]
The three examples are added in a current branch :

https://github.com/mbaudin47/openturns/commit/ef1fae0092cf74d61fe6434d6dab794f6c01a207

**Comment by mbaudin47** [comment]
The default value of Kolmogorov sampling size is still too low:

https://github.com/openturns/openturns/blob/539fc81dee231523101cd7d2a30f83fa636ba248/lib/src/Base/Common/ResourceMap.cxx#L1404

Given the improvements in #1621 and #1597, may the default sampling size be increased? This would highly improve the current situation, isn't it @regislebrun @sofianehaddad ?

**Comment by mbaudin47** [comment]
This is currently be fixed in #1621
